receiver/gitproviderreceiver/metadata.yaml

```diff
@@ -53,6 +53,41 @@ metrics:
     gauge:
       value_type: int
     attributes: [repository.name]
+  git.repository.branch.time:
+    enabled: true
+    description: Time the branch has existed.
+    unit: "{branch}"
```

Comment:
I don't think `branch` makes sense as the unit for these newly added metrics.  For this, would `s` make sense, similar to the `git.repository.pull_request.time_open` metric below?

(Same comment for the rest of the added metrics, their units should be updated as well)

Reply:
Happy to adjust. We originally had them as `s` but adjusted to ` branch` based on some change in understanding on conventions. After re-reviewing the semconv on this, I agree it should be `s`. The previous change was a misunderstanding. Thanks for the catch.

Reply:
Adjusted all metrics after re-reading the conventions.